### ISSUE TITLE
Show timer on screen and improve the timer

### ISSUE THREAD

**Issue by PvtVoid** [Talishar/Talishar-FE#510]
Here's Kobe idea from discord: https://discord.com/channels/865412150125789214/971928581267664936/1208151399292997712

> is it possible to have the timer show up on screen? or at least warn you somehow if you're running out of time (i.e. top left corner with a red notification box that says "Player1 has 15 seconds left" or shows in the game log box)
> 
> in regards to thoughts on how to improve the timer in general, i personally really like how Pokemon Showdown has their inactivity timer set up (something like this https://www.smogon.com/forums/threads/ps-timer-updates.3646406/), it prevents players from dragging out games too much but also lets people tank on important turns if needed

**Comment by LaustinSpayce** [comment]
Yeah it could work. The difficult part is the backend though. We can show timers on the front end all we want but it's the server that enforces if people are AFK or not. That's the hard part.

If you want to have a go at implementing it open a PR and send another message on discord.

**Comment by bruno-edo** [comment]
I think right now the low hanging fruit is putting a visual time
representation on the front-end. Because while playing you have no idea how
much time you've left.

This, in my opinion, is so bad that I resorted to just doing a random play
and undoing simply so the server doesn't give the "claim victory" option to
my opponent (people hit that faster than the speed of light).

Next iteration we can focus on improving the server handling of this.
